### PR TITLE
Removes the Sheet Snatcher 9000

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Removes the Sheet Snatcher 9000 and its (unused) parent.

## Why It's Good For The Game
The SS9000 is a holdover of when the ORM didn't exist and ores had to be refined on Lavaland (or so I am told). It's not really useful for the mining cyborg's job.

## Testing
Game compiled. Mining borg didn't have the SS9000.

## Changelog
:cl:
del: Removed the Sheet Snatcher 9000.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/bags.dm [hunk 1]
```diff
@@ -262,143 +261,6 @@
 		depth = min(8, depth + 1)
 		unsorted.sort(depth)
 
-// -----------------------------
-//        Sheet Snatcher
-// -----------------------------
-// Because it stacks stacks, this doesn't operate normally.
-// However, making it a storage/bag allows us to reuse existing code in some places. -Sayu
-
-/// what is this even used for
```
Comment (1080pCat) [inline code/game/objects/items/weapons/storage/bags.dm:271]: mfw

**Review by 1080pCat — APPROVED** [review]

**Comment by spacegeese** [comment]
this means borgs can't pick up sheets made by ash drake battles, which sucks because those are hard to lug around as borgs

**Comment by Fordoxia** [comment]
> this means borgs can't pick up sheets made by ash drake battles, which sucks because those are hard to lug around as borgs

Do drakes not drop a chest that you can use to haul stuff once you've killed them as a borg?

**Comment by Oyu07** [comment]
aw. I use this when I smelt at ORM and then bring some mats to robotics. 
Definetly outdated and not used. o7

**Review by Burzah — CHANGES_REQUESTED** [review]
The icon needs to be removed from `mining.dmi`

**Comment by Fordoxia** [comment]
> The icon needs to be removed from `mining.dmi`

It is done.

**Review by Burzah — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

**Pull request merged**